Tools/Logger.py

```
@@ -1,80 +1,296 @@
+        # Try to extract a value - if successful, group the message
+        try:
+            if log_msg.extract_value() is not None:
+                groups = self.context.logger_storage['message_groups']
+                if log_msg.base_hash not in groups:
+                    groups[log_msg.base_hash] = MessageGroup(log_msg)
+                else:
+                    groups[log_msg.base_hash].add_message(log_msg)
+            else:
+                daily_messages = self.context.logger_storage['daily_messages']
+                daily_messages[log_msg.hash].append(log_msg)
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)